CLA.md

```diff
@@ -1,100 +1,37 @@
-### LDTTeam CLA
+# LDTTeam CLA
```

Comment:
Just a note all of this is legal stuff, so just be careful that you didn't change any of the text or meaning of the text =D

Reply:
Does bolding some of it/italicizing some count as changing it?

Reply:
i would think it's fine. but i'm not a lawyer lol